Yosemite/Yosemite/Model/Model.swift

```diff
@@ -74,7 +74,7 @@ public typealias OrderCouponLine = Networking.OrderCouponLine
 public typealias OrderFeeLine = Networking.OrderFeeLine
 public typealias OrderFeeTaxStatus = Networking.OrderFeeTaxStatus
 public typealias OrderGiftCard = Networking.OrderGiftCard
-public typealias OrderMetaData = Networking.OrderMetaData
+public typealias OrderMetaData = Networking.MetaData
```

Comment:
Discussion: 

Later on, given that we're going to be using `Networking.MetaData` also for product, I wonder if it's good to not rename `OrderMetaData` here, and then also add

`public typealias ProductMetaData = Networking.MetaData`

Some benefits I can think of:

1. Not renaming `Yosemite.Model.OrderMetaData` means not having to update various part of the code that already uses it.
2. Consumers of Yosemite do not have to care or know that order and product both share the same  `Networking.MetaData` underneath.

Not sure what the downsides are for this approach. What do you think?

Reply:
This is an interesting take. I was concerned about the initialization of `OrderMetaData` in unit tests, but it looks like the compiler can still accept the initialization of the underlying type. Let's keep this for now and we can change later if we catch any issue with it.